week2/advanced-js/advanced-js-solution.md

```diff
@@ -0,0 +1,118 @@
+# Advanced JavaScript solutions
+
+This is the solution for the advanced JavaScript workshop, only look at it in case you have solved the question and want to check different solutions.
+
+1. Format a string
+
+  Your task was to loop over the object keys and return the value concatenated
+
+  Mario's solution
+
+  ```js
+  function list(names) {
+    let sentance = "";
```

Comment:
I notice here you've used `let` and `const`, but for the other solutions you have used `var`. It probably makes sense to change this to `var` right, as they haven't encountered ES6 yet?

Reply:
I'll change it, it's a copied solution from codewars :smile: